ci-operator/step-registry/rosa-qe/sts/cluster/nightly/rosa-qe-sts-cluster-nightly-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: rosa-qe-sts-cluster-nightly
```

Comment:
1. Avoid to use `qe` text in workflow name. 
2. why use _nightly_ here? is it only for nightly build?

Reply:
yes, only for the nightly build.